chain/client/src/stateless_validation/partial_witness/partial_witness_actor.rs

```diff
@@ -78,7 +79,7 @@ pub struct DistributeStateWitnessRequest {
     pub epoch_id: EpochId,
     pub chunk_header: ShardChunkHeader,
     pub state_witness: ChunkStateWitness,
-    pub contract_deploys: Vec<ContractCode>,
+    pub contract_updates: ContractUpdates,
```

Comment:
now passing both accesses and deploys from client actor to partial witness actor in DistributeStateWitnessRequest .